scipy/io/wavfile.py

```diff
@@ -470,7 +470,7 @@ def _read_data_chunk(fid, format_tag, channels, bit_depth, is_big_endian, is_rf6
             count = size if dtype == 'V1' else n_samples
             data = np.fromfile(fid, dtype=dtype, count=count)
         except io.UnsupportedOperation:  # not a C-like file
-            fid.seek(start, 0)  # just in case it seeked, though it shouldn't
+            fid.seek(start, 0)  # just in case it sought, though it shouldn't
```

Comment:
can we revert this change (it is about talking about `seek`)

Reply:
Seeked vs Sought: Sought is the correct English word that is the past tense of ‘Seek’. Seeked is an incorrect word and not present in the English dictionary. “Seeked” is a misspelling word of sought. (https://www.aplustopper.com/seeked-vs-sought/)

But you are of course right, with the "correct" spelling the reference to the function name becomes less clear.